test/unit_tests/utils/process-json_test.js

```diff
@@ -247,4 +247,47 @@ describe( 'process-json', function() { // eslint-disable-line max-statements, no
 
   } );
 
+  describe( 'processDelinquencies', function() {
+
+    var data = [
+      {
+        meta: {
+          name: 'foo'
+        },
+        data: [ {
+          date: 1234,
+          pct30: 5678,
+          pct90: 9012
+        } ]
+      },
+      {
+        meta: {
+          name: 'bar'
+        },
+        data: [ {
+          date: 12342,
+          pct30: 56782,
```

Comment:
this kind of data is perfect - makes it easier for me to understand what the test is for 👍